utils/wasm-gen/src/config.rs

```diff
@@ -182,6 +195,12 @@ impl<T: Into<Hash>> ConfigsBundle for StandardGearWasmConfigsBundle<T> {
 
         let selectable_params = SelectableParams::default();
 
+        let mut injection_types = injection_types;
+        if critical_gas_limit.is_some() {
+            injection_types
+                .enable_syscall_import(InvocableSysCall::Loose(SysCallName::GasAvailable));
```

Comment:
I believe that must be done on a lower level when defining value in `GearWasmGeneratorConfigBuilder`.

Reply:
done in d793944027c1b23af747be9f3ded0bb7f6562990